ts-packages/web/src/components/feed-card.tsx

```
@@ -175,13 +180,14 @@ export function FeedContents({
   url?: string;
 }) {
+  const { theme } = useTheme();
   const c =
     typeof window !== 'undefined' ? DOMPurify.sanitize(contents) : contents;
 
   return (
     <Col className="text-white">
       <p
-        className="feed-content font-normal text-[15px]/[24px] align-middle tracking-[0.5px] text-c-wg-30 px-5"
+        className={`feed-content font-normal text-[15px]/[24px] align-middle tracking-[0.5px] px-5 ${theme === 'light' ? 'text-neutral-600' : 'text-c-wg-30'}`}
         dangerouslySetInnerHTML={{ __html: c }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)